x-pack/plugins/notifications/server/services/email.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { UnsecuredActionsClient } from '@kbn/actions-plugin/server/unsecured_actions_client/unsecured_actions_client';
+import type { IEmailService, PlainTextEmail } from './email_service';
+
+export class EmailService implements IEmailService {
+  constructor(
+    private requesterId: string,
+    private connectorId: string,
+    private actionsClient: UnsecuredActionsClient
+  ) {}
+
+  async sendPlainTextEmail(params: PlainTextEmail): Promise<void> {
+    const actions = params.to.map((to) => ({
+      id: this.connectorId,
+      spaceId: 'default', // TODO should be space agnostic?
```

Comment:
@ymao1 I'm still not clear on the values needed  `spaceId` and `executionId`.
From what I understood case assignment (and therefore the notifications mechanism) should be _space_ agnostic. I think that the idea is to use shared SOs.